docs/gunicorn_ext.py

```diff
@@ -39,7 +39,7 @@ def format_settings(app):
     known_settings = sorted(guncfg.KNOWN_SETTINGS, key=lambda s: s.section)
     for i, s in enumerate(known_settings):
         if i == 0 or s.section != known_settings[i - 1].section:
-            ret.append("%s\n%s\n\n" % (s.section, "-" * len(s.section)))
+            ret.append("{}\n{}\n\n".format(s.section, "-" * len(s.section)))
```

Comment:
%-formatting will stay forever in Python, so I don't these changes are in the scope of this PR.

Reply:
Done by reverting 159aa1ed8ec0455eadc2ab029bd9ef670782539c.